wp-content/themes/chrisdinh-photography/tailwind.config.cjs

```diff
@@ -17,6 +17,13 @@ module.exports = {
         md: '768px',
         lg: '1024px',
         xl: '1440px'
+      },
+      padding: {
+        '1/2': '50%',
```

Comment:
Remove both 1/2 and full if we're not using them

Reply:
updated in commit 7a8dd48d77f9683e02904cbb06e5dc28970869e9